### PR TITLE
jdbi.org favicon

### PR DESCRIPTION
**Pull request by TheRealMarnes**
Everybody deserves to look nice :)

image

asciidoctor needs to see a doctor:

image

### CODE REVIEW

File: docs/src/adoc/index.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 :source-highlighter: coderay
 :source-language: asciidoc
 :imagesdir: images
+:docinfo: private
```
Comment (stevenschlansker) [inline docs/src/adoc/index.adoc]: I'm not sure I understand this change.  Is this not the default? 

https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/docinfo.adoc

> Setting docinfo with no value is equivalent to setting the value to private.

Comment (TheRealMarnes) [reply docs/src/adoc/index.adoc]: done

Comment (stevenschlansker) [reply docs/src/adoc/index.adoc]: Can we not omit the key entirely?  (That was the change I expected to see)  Or is its presence important?

Comment (TheRealMarnes) [reply docs/src/adoc/index.adoc]: I tried it, doesn't work. The doc implies as much.

**Comment by TheRealMarnes** [comment]
I'll just hotfix that then, preferred it too

**Pull request merged**